core/README.md

```diff
@@ -1,22 +1,149 @@
+  <source media="(prefers-color-scheme: light)" srcset="../assets/Product_Icons_light/launch-light.svg" width="14.0%">
+  <img alt="Weights and Biases Launch" src="">
+</picture>
+</a>
+<a target="_blank" href="https://docs.wandb.ai/guides/model_registry?utm_source=github&utm_medium=code&utm_campaign=wandb&utm_content=readme">
+<picture>
+  <source media="(prefers-color-scheme: dark)" srcset="../assets/Product_Icons_dark_background/model-registry-dark.svg" width="14.0%">
+  <source media="(prefers-color-scheme: light)" srcset="../assets/Product_Icons_light/model-registry-light.svg" width="14.0%">
+  <img alt="Weights and Biases Model Management" src="">
+</picture>
+</a>
+<a target="_blank" href="https://docs.wandb.ai/guides/artifacts/project-scoped-automations?utm_source=github&utm_medium=code&utm_campaign=wandb&utm_content=readme">
+<picture>
+  <source media="(prefers-color-scheme: dark)" srcset="../assets/Product_Icons_dark_background/automations-dark.svg" width="14.0%">
+  <source media="(prefers-color-scheme: light)" srcset="../assets/Product_Icons_light/automations-light.svg" width="14.0%">
+  <img alt="Weights and Biases Prompts" src="">
+</picture>
+</p>
+
+See the [W&B Developer Guide](https://docs.wandb.ai/?utm_source=github&utm_medium=code&utm_campaign=wandb&utm_content=documentation) and [API Reference Guide](https://docs.wandb.ai/ref?utm_source=github&utm_medium=code&utm_campaign=wandb&utm_content=documentation) for a full technical description of the W&B platform.
+
+# `wandb-core`: A new backend for the W&B SDK
 
 ## Introduction
 
 `wandb-core` is a new and improved backend for the W&B SDK that is more performant, versatile, and robust.
 
-## Getting started
+**Upgrade now and experience:**
+
+* 🚀 **Logging performance:** Experience up to 88% performance improvements when logging with multiple processes.
+* 🤏 **Reduced resource consumption:** Lower memory footprint allows you to run more experiments on your machines.
+* 📊 **Improved Table logging:** Experience up to 40% faster table logging performance! 
+* ⚡ **Faster startups & shutdowns:** Enjoy up to 36% faster startup and shutdown times.  
+* ⏩ **Enhanced artifact handling:** Experience up to 33% faster Artifact uploads speed and up to 27% faster Artifact retrieval!
+* 🌐 **Faster offline sync:** Keep your long-running experiments synced effortlessly with improved offline speeds.
+
+See our (benchmark analysis)[https://github.com/wandb/wandb/blob/main/tools/bench/bench.md] for more information on performance improvements.
+
+## Quickstart
+Get started with wandb-core in four steps:
+
+1. First, sign up for a [free W&B account](https://wandb.ai/login?utm_source=github&utm_medium=code&utm_campaign=wandb&utm_content=quickstart).
+
+2. Second, install the W&B SDK with [pip](https://pip.pypa.io/en/stable/). Navigate to your terminal and type the following command:
+
+```bash
+pip install wandb
+```
+***Note: ensure you have `wandb>=0.17.0`.***
```

Comment:
Let's update this to the latest version and add this file to the bumpversion config.

Reply:
Updated. But not sure how to include a bumpversion config to maintain it automatically

Reply:
add an entry for the readme file here: https://github.com/wandb/wandb/blob/main/.bumpversion.wandb.cfg

Reply:
Ok: https://github.com/wandb/wandb/pull/7919